app/javascript/lib/shared/modules/module-search.js

```diff
@@ -60,73 +63,42 @@ $(document).on('turbolinks:load', function() {
     }
   }
 
-  function searchFrontCallback(err, content) {
-    if (err) {
-      console.error(err);
-      return;
-    }
-
-    $resultsContainer.html('');
-    var sum = 0;
-    content.results.forEach(function(indexResults){
-      sum += indexResults.nbHits;
-    });
-
-    if(sum > 0) {
-      content.results.forEach(function(indexResults){
-        indexResults.hits.forEach(function(d){
-
-          var result = '<a class="result" <a href="' + d.resource_path + '">' +
-            '<h2>' + (d['title'] || d['name'] || d['title_' + I18n.locale] || d['name_' + I18n.locale]) + '</h2>' +
-            '<div class="description">' +
-              '<div>' + itemDescription(d) + '</div>' +
-              '<span class="soft item_type">' + itemType(d) + '</span>' +
-              (itemUpdatedAt(d) ? ' · <span class="soft updated_at">' + itemUpdatedAt(d) + '</span>' : '') +
-            '</div>' +
-          '</a>';
-
-          var div = $(result);
-          div.appendTo($resultsContainer);
-        });
-      });
-    } else {
-      $('<div class="result"><p>'+I18n.t("layouts.search.no_results")+'</p></div>').appendTo($resultsContainer);
-    }
-
-    if(window.searchClient.indexes.length > 1) {
-      $('<div class="result"><small>'+I18n.t("layouts.search.powered_by")+'</small></div>').appendTo($resultsContainer);
-    }
+  function formattedResult(d){
+    return '<h2>' + d['title'] + '</h2>' +
```

Comment:
You can use interpolations, i.e `<h2>${d['title']}</h2>` + ....

Reply:
Done!